app/src/main/res/layout/store_activity.xml

```diff
@@ -35,6 +35,7 @@
         android:layout_width="match_parent"
         android:layout_height="@dimen/tabsHeight"
         android:background="?attr/backgroundTabs"
+        android:elevation="4dp"
```

Comment:
after talking with Francisco about this and testing in a device, I think it would be a better solution to remove both this and toolbar's 4dp to have them both at the same level. It doesn't look like it is affecting anything in other screens. I think it is a better approach that could save some confusion in the future if we have to mess with elevations again.